src/components/add-ticket/AddTicketForm.tsx

```diff
@@ -56,6 +57,11 @@ export const AddTicketForm = () => {
       active: tag === TagConstIds.transport,
       onClick: () => setTag(TagConstIds.transport),
     },
+    {
+      name: i18n.addTicket.wizard.formNameHousing,
+      active: tag === TagConstIds.housing,
+      onClick: () => setTag(TagConstIds.housing),
```

Comment:
while ur here, could you please adjust font size on mobile to make it bigger and possibly wrap elements in second picker?

<img width="400" alt="Screenshot 2022-03-10 at 13 26 10" src="https://user-images.githubusercontent.com/25708300/157661573-28da7e9c-e619-4ca2-b745-3e8691bd2444.png">

Reply:
How do you feel about this?

<img width="688" alt="CleanShot 2022-03-10 at 14 00 16@2x" src="https://user-images.githubusercontent.com/1051509/157666947-cdde7b5f-d323-4bf1-b96c-8689b9c056b5.png">


Would you still like them to wrap?

Reply:
Depends how it looks on UA, but here looks ok!